<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>889</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RGB</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../pyqt/rgb_icon.png</normaloff>../pyqt/rgb_icon.png</iconset>
  </property>
  <widget class="QPushButton" name="pushButton_open">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>450</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>이미지 선택</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_picture">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>421</width>
     <height>421</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextBrowser" name="rgb_text">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>230</y>
     <width>411</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>459</x>
     <y>10</y>
     <width>411</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="graph_verticalLayout"/>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>450</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>현재 파일 :</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QTextBrowser" name="filename">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>450</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
